modules/RTC/TraceablePeerConnection.js

```diff
@@ -315,14 +315,15 @@ export default function TraceablePeerConnection(
         this.peerconnection.onremovestream
             = event => this._remoteStreamRemoved(event.stream);
     } else {
-        this.peerconnection.ontrack = event => {
-            const stream = event.streams[0];
+        this.onTrack = evt => {
```

Comment:
why this change?

Reply:
To avoid running into issues when something else like callstats/rtc stats accidentally overrides these methods.